### DISCUSSION TITLE
is Storybook v5.2.4 compatible with webpack 5

### DISCUSSION THREAD

**Discussion by krishmurarka** [storybookjs/storybook#18547] [Help] [answered]
Hi,
Currently, in our application, we are using storybook v5.2.4 and webpack4.

So we were planning on upgrading our web pack 4 to web pack 5, so will webpack5 be still compatible with our current storybook version, or we would need to first bump our storybook ??

**Comment by shilman** [comment, marked as answer]
The first version to support experimentally is Webpack5 is 6.2, and your best bet is to upgrade to the latest stable version, 6.5.x.

Since the upgrade from SB5 to SB6 is a breaking change, we've prepared a guide here: https://storybook.js.org/blog/storybook-6-migration-guide/
